autotest/ogr/ogr_flatgeobuf.py

```diff
@@ -416,6 +416,20 @@ def test_ogr_flatgeobuf_datatypes():
     assert f['string'] == 'my string'
     assert f['datetime'] == '2019/10/15 12:34:56.789+00'
 
+def test_ogr_flatgeobuf_mixed():
+    ds = ogr.Open('data/testfgb/testmixed.geojson')
+    lyr = ds.GetLayer(0)
+    f1 = lyr.GetNextFeature()
+    f2 = lyr.GetNextFeature()
+
+    ds = ogr.GetDriverByName('FlatGeobuf').CreateDataSource('/vsimem/test.fgb')
```

Comment:
you could likely use gdal.VectorTranslate() to simplify this a bit. And this test doesn't check the reading side

Reply:
Thanks for the tip, test is rewritten now.